doc/gendoc.py

```diff
@@ -656,6 +672,20 @@ def stream_consume_member_initializers(self, classname, stream):
                 self.objInfo.member_init(classname, arg1.value, init_list)
             else:
                 stream.pop()
+        # we're now on a codeblock
+        self.stream_consume_setDoc(classname, TokenStream(codeblock.value.enclosed_tokens))
+
+    def stream_consume_setDoc(self, classname, stream):
```

Comment:
Should test_gendoc.py be extended for the whole doc parsing thing?

Reply:
There is no way to obtain the doc string from a running hlwm instance, so we can't test it. Once there is a `help` command, I planned to test it.

Reply:
I see, thanks!